modules/ROOT/pages/asm-secret-group-concept.adoc

```diff
@@ -5,33 +5,40 @@ endif::[]
 
 == Availability and Consistency
 
-By implementing Edit Sessions every time a user needs to edit secrets inside a secret group, Secrets Manager guarantees service of a consistent set of secrets.
+Secrets Manager implements edit sessions every time a user needs to edit secrets inside a secret group to guarantee service of a consistent set of secrets.
 
-Consider the scenario where you need to renew secrets inside a secret group: You need to click the Edit button in the Anypoint Platform UI. This opens an edit session during which the secret group locks out other users, and you can edit all the necessary security objects inside the secret group. While the edit session is open, Secrets Manager serves the same secrets it had before you opened the edit session. This guarantees that any redeployed application would still see the same, consistent secret group even though you are modifying it.
+For example, if you need to renew secrets inside a secret group, you click the *Edit* button in the Secrets Manager UI. This opens an edit session and locks out other users while you edit the necessary security objects inside the secret group. 
+
+While the edit session is open, Secrets Manager serves the same secrets it served before you opened the edit session. This guarantees that any redeployed application still sees the same secret group even while you are modifying it.
 
 image::asm-secret-group-concept-cd233.png[]
 
-After you complete your changes, you must click the Finish button in the Anypoint Platform UI. This action updates the secret group with all the renewed secrets that Secrets Manager would serve to all redeployed applications after that point.
+After you complete your changes and click the *Finish* button, the secret group is updated with all the updated secrets that Secrets Manager will serve to all redeployed applications after that point.
 
 image::asm-secret-group-concept-8fc8a.png[]
 
 == Concurrency
 
-When another user is editing a secret group, the Anypoint Platform UI provides a Cancel Edit button next to the secret group, along with the name of the user who is editing the group.
+When a user edits a secret group, the name of the user who is currently editing the secret group is displayed. 
 
 image::asm-secret-group-concept-4876b.png[]
 
-If the editing user forgets about an open edit session and is not available, you can click the Cancel Edit button to clear the that user's session. Note that this action deletes all that user's changes, but it allows you to click the now available Edit button to open a new edit session.
+If the user who is editing the secret group leaves the edit session open and is not available to cancel the edit session, you can click the *Cancel Edit* button to clear that user's session. 
+
+[NOTE]
+If you cancel a user's edit, it deletes all that user's changes. 
+
+The *Edit* button is then available for you to open a new edit session. 
 
```

Comment:
Just an overall note that I'm surprised to see this level of detail in a conceptual topic.

Reply:
Yeah, that's the way I inherited it. I would have made this a task.